readthedocs/doc_builder/backends/sphinx.py

```diff
@@ -170,6 +174,7 @@ def get_config_params(self):
             'downloads': downloads,
             'subproject_urls': subproject_urls,
             'build_url': build_url,
+            'vcs_url': vcs_url,
```

Comment:
Why don't we pass always `self.version.vcs_url` instead only when it's external? I think that will simplify code reading and thinking about _why_.

Otherwise, we could have a comment in the previous if.

Reply:
this is only used in external versions currently